src/Riak/Command/Indexes/Query.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+/*
+Copyright 2014 Basho Technologies, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+namespace Basho\Riak\Command\Indexes;
+
+use Basho\Riak\Command;
+use Basho\Riak\CommandInterface;
+
+/**
+ * Class Query
+ *
+ * Riak 2i query information.
+ *
+ * @author Alex Moore <amoore at basho d0t com>
+ */
+class Query extends Command\Object implements CommandInterface
```

Comment:
This may be a copy/paste thing, but I don't think that this Command shares enough in common with the Object commands to be extending Command\Object. I think it should extend Command instead.